manifests/cli.pp

```diff
@@ -36,10 +35,11 @@
     creates => $jar,
   }
   ~> exec { 'jenkins-cli' :
-    command     => "${extract_jar} && ${move_jar} && ${remove_dir}",
+    command     => "${download_jar} && ${move_jar}",
     path        => ['/bin', '/usr/bin'],
     cwd         => '/tmp',
     refreshonly => true,
+    onlyif      => lass['jenkins::service'],
```

Comment:
Did you intend the require here? There's also a typo in `Class`. Note that you also need to be aware that the class is not always defined (`$manage_service => false`).

Reply:
Yeh i basically just used the one we use for our servers, but noticed it will fail with manage-service and havent had time to figure out how to fix it.

If you have a solution feel free to edit